git-ftp

```diff
@@ -76,6 +76,7 @@ declare -i ENABLE_REMOTE_LCK=0
 declare -i ACTIVE_MODE=0
 declare -i USE_KEYCHAIN=0
 declare -i EXECUTE_HOOKS=1
+declare -i DISABLE_PPE=1
```

Comment:
Anyone else looking into the code later, will not know what `DISABLE_PPE` means. I suggest using a variable name similar to the command line option like `ENABLE_POST_ERRORS=0`.

Reply:
I'm okay with that : i had to choose between a short or long name, I went for the short one as a beginning.
Yet I have a question : as this was a non-backward compatible change, I've opted for a dev that was not changing the behavior of *your* program (and what others are expecting from it).
Was I right to keep backward-compatible behavior or shoud we ditch this option and always take care of the post script return code ?

Reply:
Backwards compatible is great. We can then change the behaviour in the future if necessary and ditch the variable if nobody uses it any more.